File: PROYECTO/PSport-APP/PSport/Model/DTOs/UsuarioRegister.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PSport.Model.DTOs
{
    public class UsuarioRegister
    {
        public string dni { get; set; }
        public string nombre { get; set; }
        public string apellido1 { get; set; }
        public string apellido2 { get; set; }
        public string fechaNacimiento { get; set; }
        public string email { get; set; }

        //sha512
        public string password { get; set; }
    }
}

File: PROYECTO/old/PP-Sport-API-/PPSportAPI/src/main/kotlin/es/sergigavi/PPSportAPI/MODEL/REQUEST/UsuarioRegister.kt
package es.sergigavi.PPSportAPI.MODEL.REQUEST

import lombok.*
import java.time.LocalDate

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
data class UsuarioRegister(

    var dni:String,

    var nombre:String,

    var apellido1:String,

    var apellido2:String?,

    var fechaNacimiento: LocalDate,

    var email:String,

    var password:String,
)
{
}